docs/Project.toml

```diff
@@ -1,3 +1,6 @@
 [deps]
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Meshes = "eacbb407-ea5a-433e-ab97-5258b1ca43fa"
 MeshIntegrals = "dadec2fd-bbe0-4da4-9dbe-476c782c8e47"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
```

Comment:
Would be good to also have a compat entry in the docs Project.toml for all dependencies (except MeshIntegrals.jl itself).